### ISSUE TITLE
Contact Center Review for form 22-10203 - STEM Scholarship Form

### ISSUE THREAD

**Issue by dneel-bah** [department-of-veterans-affairs/va.gov-team#12670]
# Request the Contact Center Review 

## Description
Edith Nourse Rogers STEM Scholarship Application (VA Form 22-10203)
The Rogers STEM Scholarship will provide up to nine months of additional Post-9/11 GI Bill benefits (to a maximum of $30,000) to qualifying Veterans and Fry Scholars seeking an undergraduate STEM degree or who have earned a STEM degree and are seeking a teaching certification.

### Launch Date
Targeting Limited Rollout Phase 1 08/26/20 - 08/30/20
Targeting Limited Rollout Phase 2 09/01 - 09/03/20

During the limited rollout a subset of the applicants will be routed to the new 10203 form while the remaining applicants continue to be routed through the 1995.

Targeting Production Launch between 08/31/20 - 09/11/20

### Product Guide
[10203_ProductGuide_08202020.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5104505/10203_ProductGuide_08202020.docx)
[10203_ProductGuide_08202020.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/5104506/10203_ProductGuide_08202020.pdf)

### Product Video
[STEM-10203-Call-Center-Demo.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/5098794/STEM-10203-Call-Center-Demo.zip)

### Product Video Transcript
[STEM-10203-Call-Center-Transcript.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5098780/STEM-10203-Call-Center-Transcript.docx)

## Acceptance Criteria
- [ ] VSP Analytics & Insights QA has been completed 
- [ ] VSP Analytics & Insights sends artifacts to appropriate contact centers
- [ ] VFS team completes brief VSP Collaboration Cycle Feedback survey [link](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before)

**Comment by Kimberley2019** [comment]
@dneel-bah Thank you, we will review and send to the contact center for review and feedback.

**Comment by dneel-bah** [comment]
@Kimberley2019 The routing of the applicant to the 1995 vs the 10203 is just for the limited production rollout time period.  We are estimating about 1 week for that.

Based on historical submissions during the last few months, there are expected to be ~500 applications submitted per week.

There will be 2 phases of rollout.  During the initial phase 10% of the applicants will be routed to the 10203, which will be ~50 applicants.  During the second phase, 30% of the applicants will be routed to the 10203 (~150 applicants).  All other applicants will continue to be routed to the 1995 with the STEM scholarship functionality.

These 2 phases are expected to last about a week before 100% of the applicants are routed to the 10203.  Once that happens, the 1995 will have the STEM questions removed.

Regarding these questions:
**Is there a time frame for the application period?** Can you elaborate?
**Is this a scholarship that will be returning each year?** Yes, I think there are details about renewal and caps that I will need to get from EDU.
**Is this a rolling scholarship?** Can you elaborate?

**Comment by Kimberley2019** [comment]
@dneel-bah Thank you for the explanation of the routing, the Contact Center asked for that clarification so that they could be prepared for any possible questions concerning the different forms.  It seems as though they may not receive any questions about the routing of the forms.  I will pass this information along to them today and ask if further details are needed.

For the second set of questions, one of the Contact Center team members were trying to understand if there was a time frame for applications to be submitted.  Are they only accepting applications from 9/1/2020 to 12/31/2020?

**Comment by dneel-bah** [comment]
@Kimberley2019 EDU provided this link for information regarding the scholarship https://benefits.va.gov/gibill/fgib/stem.asp.  On a call today EDU mentioned that there are some policy changes in the works that may change the benefit.  


Additionally, some others at DEPO are working on some static content pages that will be available soon that you can preview here;
· Static Page - http://preview-prod.vfs.va.gov/preview?nodeId=6860
· Other Benefits page - http://preview-prod.vfs.va.gov/preview?nodeId=945

**Comment by Kimberley2019** [comment]
@dneel-bah Thank you for passing this information along to us.  I am able to review the link for scholarship information, however, I am unable to view the two additional links.  The message below is what I am receiving for both, are you able to verify that the links are correct and active so that I can pass it along?

This site can’t be reachedpreview-prod.vfs.va.gov’s server IP address could not be found.
Try running Windows Network Diagnostics.
DNS_PROBE_FINISHED_NXDOMAIN

Will the policy changes that were mentioned on the call have any affect on the product information that we have provided to the Contact Center?  If so, will the items that have previously been sent need to be updated?

Also, there is no information regarding a time frame for the scholarship, therefore I am assuming that this will be ongoing.  Can you please confirm that is correct.

**Comment by dneel-bah** [comment]
@Kimberley2019 

The provided links are only available on the VA network.  They are "preview" pages that will be live in the near future.

Regarding the policy changes, I talked with EDU today and they said that they would pass along any updated information to the call center.  At this time, they do not see it having an impact on the form or submission process, it will only be on the processing that goes on behind the scenes.

I also confirmed with EDU that there is no expiration date for the scholarship program.
